spec/rental_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe Rental do
+  before :each do
+    @book = Book.new 'Title', 'Author'
+    @person = Teacher.new 26, 'Specialization', 'Name'
+    @rental = Rental.new 'Date', @book, @person
+  end
+
+  describe '#new' do
+    it 'takes three parameters and returns a rental object' do
+      @rental.should be_an_instance_of Rental
+    end
+  end
+
+  describe '#date' do
+    it 'returns the correct date' do
+      @rental.date.should eql 'Date'
+    end
+  end
+
+  describe '#person' do
+    it 'returns the correct person' do
+      @rental.person.age.should eql 26
+      @rental.person.specialization.should eql 'Specialization'
+      @rental.person.name.should eql 'Name'
+    end
+  end
+
+  describe '#book' do
+    it 'returns the correct book' do
+      @rental.book.title.should eql 'Title'
+      @rental.book.author.should eql 'Author'
+    end
+  end
+end
```

Comment:
- [ ] In the `Rental` class, you have these lines of codes:

```
  person.rentals.push(self)
  book.rentals.push(self)
```

#### Kindly test if they correctly push in a new instance of the `Rental` object. 👍

Reply:
Thank you, we resolved this

Reply:
Thank you, we have resolved this